pyamg/multilevel.py

```diff
@@ -170,6 +171,9 @@ def __init__(self, levels, coarse_solver='pinv2'):
         for level in levels[:-1]:
             if not hasattr(level, 'R'):
                 level.R = level.P.H
+            if not hasattr(level, 'auxiliary'):
```

Comment:
@436ahsan 

I see that there is some code related to gen_air here.  Will this be a full or partial implementation of gen_air in master_air?  I ask because we probably don't want to have a partial implementation, and if we want to port a full implementation of gen_air into master_air, then we may want to ask Ben.

If you want to have your own repository that combines master and gen_air, you can always have your own fork that does this.  

Let me know.

Reply:
ya no gen air in master air right now, remove any gen air stuff from the pr

Reply:
@436ahsan Hi Ahsan, if you remove the gen air changes, then I think this should be ready to merge.

If you haven't considered this already, you could maintain your own fork of master_air with gen_air merged inside it.  This would be your own git repo, with version history, and all the data backups that come with that. This could be a good solution.